src/components/Channel/Channel.js

```diff
@@ -192,10 +107,7 @@ const ChannelInner = ({
           mainChannelUpdated = false;
         }
 
-        if (
-          mainChannelUpdated &&
-          e.message.user.id !== chatContext.client.userID
-        ) {
+        if (mainChannelUpdated && e.message.user.id !== client.userID) {
```

Comment:
we need to get rid of this `e` single letter variable naming pattern, it's not best practice